src/sentry/search/events/fields.py

```diff
@@ -2292,6 +2293,7 @@ def __init__(
         self,
         name: str,
         allowed_columns: Optional[Sequence[str]] = None,
+        allow_custom_measurements: Optional[bool] = False,
```

Comment:
🤔 looking at the changes in `src/sentry/search/events/datasets/metrics.py` should we default to True instead?

Reply:
👍 done!